tools/Microsoft.AspNetCore.Server.Kestrel.GeneratedCode/KnownHeaders.cs

```diff
@@ -366,10 +384,19 @@ protected override bool RemoveFast(string key)
             return MaybeUnknown?.Remove(key) ?? false;
         }}
         protected override void ClearFast()
-        {{
-            _bits = 0;
-            _headers = default(HeaderReferences);
+        {{            
             MaybeUnknown?.Clear();
+            {Each(loop.Headers.OrderBy(h => !h.PrimaryHeader), header => $@"
+                if ({header.TestBit()})
+                {{
+                    _headers._{header.Identifier} = default(StringValues);
```

Comment:
I haven't tested that change yet, but moving to 1.1.0 looks like a fairly consistent 1.5-2% win.  Since JIT_Memset was showing up in the profiles for ClearFast that could be a win for the current implementation.